src/peft/__init__.py

```diff
@@ -68,4 +68,5 @@
     prepare_model_for_kbit_training,
     set_peft_model_state_dict,
     shift_tokens_right,
+    infer_device,
```

Comment:
Do we need it here? I think it's enough to have it in utils. I think we should only place functions into `peft.__init__` that we expect to be widely used by users.

Reply:
Makes sense to me, modified. Can you re-trigger CI?
Thanks

Reply:
CI re-triggered

Reply:
Fixed the test, can you re-trigger again thanks